src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {Story} from '@storybook/react'
+import {CheckboxProps} from './Checkbox.types';
+import {Checkbox, Typography} from '~/components';
+
+export default {
+    title: 'Components/Checkbox',
+    component: Checkbox,
+    parameters: {
+        layout: 'centered',
+        knobs: {disabled: true},
+        storysource: {disabled: true}
+    }
+};
+
+const Template: Story<CheckboxProps> = args => <Checkbox {...args}/>;
+const FieldTemplate: Story = ({checkboxArgs, typographyArgs}) => (
+    <>
+        <Checkbox {...checkboxArgs}/>
+        <Typography {...typographyArgs}/>
+    </>
+);
+
+export const Checked = Template.bind({});
+Checked.args = {
+    defaultSelected: true
+};
+
+export const Indeterminate = Template.bind({});
+Indeterminate.args = {
+    defaultSelected: true,
+    isIndeterminate: true
```

Comment:
I have an issue with this story. When I hover the checkbox it becomes unchecked. Maybe a CSS issue?

Reply:
Weird! I was developing in Firefox and didn't see that issue but I now do in Chrome...will fix.

Reply:
Normally I fixed it. It was because on Chrome the indeterminate doesn't trigger the pseudo-class `:checked`, now we use the attribute `aria-checked` to manage the style.